app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Login or create a new account</h1>
+
+<%= form_for(:session, url: login_path) do |f|  %>
+<%= f.label :username %>
+<%= f.text_field :username %>
+<%= f.label :password %>
+<%= f.password_field :password %>
+<%= f.submit "Login" %>
```

Comment:
we could consider adding a class to this button (and other buttons throughout the app) to make styling look snazzy.

https://v4-alpha.getbootstrap.com/components/buttons/